src/Bigcommerce/Api/Client.php

```diff
@@ -1045,6 +1045,17 @@ public static function getCustomerAddresses($id)
     }
 
     /**
+     * A single customer group by given id.
+     *
+     * @param int $id group id
+     * @return Resources\CustomerGroup
+     */
+    public static function getGroup($id)
```

Comment:
Stylistic but it seems like it would be best to call this `getCustomerGroup` for consistency with the methods above. After that, I think this can be merged.